<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>596</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>736</width>
        <height>478</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="customerSearch">
       <attribute name="title">
        <string>Search customer</string>
       </attribute>
       <widget class="QLabel" name="customerFirstName">
        <property name="geometry">
         <rect>
          <x>315</x>
          <y>20</y>
          <width>66</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>First Name</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="customerIDSearch">
        <property name="geometry">
         <rect>
          <x>25</x>
          <y>46</y>
          <width>125</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="searchCustomerSearchButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>410</y>
          <width>85</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="customerFirstNameSearch">
        <property name="geometry">
         <rect>
          <x>314</x>
          <y>46</y>
          <width>125</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="customerNameSearch">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>46</y>
          <width>125</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QDateEdit" name="dateBeginning">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>45</y>
          <width>100</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="customerID">
        <property name="geometry">
         <rect>
          <x>25</x>
          <y>20</y>
          <width>76</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Customer ID</string>
        </property>
       </widget>
       <widget class="QLabel" name="to">
        <property name="geometry">
         <rect>
          <x>565</x>
          <y>45</y>
          <width>16</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>To</string>
        </property>
       </widget>
       <widget class="QPushButton" name="searchCustomerDeleteButton">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>410</y>
          <width>91</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="dateEnding">
        <property name="geometry">
         <rect>
          <x>586</x>
          <y>45</y>
          <width>100</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="customerName">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>20</y>
          <width>36</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
       <widget class="QTableView" name="customerView">
        <property name="geometry">
         <rect>
          <x>25</x>
          <y>86</y>
          <width>658</width>
          <height>321</height>
         </rect>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
       <widget class="QLabel" name="datesInterval">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>20</y>
          <width>117</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Select time interval</string>
        </property>
       </widget>
       <widget class="QPushButton" name="searchCustomerEditButton">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>410</y>
          <width>91</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="employees">
       <attribute name="title">
        <string>Employees</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QPushButton" name="showEmployeesRefreshButton">
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item row="1" column="1">
           <widget class="QPushButton" name="showEmployeesDeleteButton">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="showEmployeesEditButton">
            <property name="text">
             <string>Edit</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QTreeView" name="treeView"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="planning">
       <attribute name="title">
        <string>Schedule</string>
       </attribute>
       <widget class="QDateEdit" name="dateEdit">
        <property name="geometry">
         <rect>
          <x>24</x>
          <y>45</y>
          <width>110</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>25</x>
          <y>20</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Planning Day</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddingCustomer"/>
   <addaction name="actionAddingEmployee"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>736</width>
     <height>22</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuAdd">
     <property name="title">
      <string>Add</string>
     </property>
     <addaction name="actionAddingCustomer"/>
     <addaction name="actionAddingEmployee"/>
    </widget>
    <addaction name="menuAdd"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menu"/>
  </widget>
  <action name="actionAddingEmployee">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/personnel.png</normaloff>:/images/personnel.png</iconset>
   </property>
   <property name="text">
    <string>Employee</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionAddingCustomer">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/client.png</normaloff>:/images/client.png</iconset>
   </property>
   <property name="text">
    <string>Customer</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>customerIDSearch</tabstop>
  <tabstop>customerNameSearch</tabstop>
  <tabstop>customerFirstNameSearch</tabstop>
  <tabstop>dateBeginning</tabstop>
  <tabstop>dateEnding</tabstop>
  <tabstop>searchCustomerSearchButton</tabstop>
  <tabstop>customerView</tabstop>
  <tabstop>searchCustomerEditButton</tabstop>
  <tabstop>searchCustomerDeleteButton</tabstop>
  <tabstop>showEmployeesEditButton</tabstop>
  <tabstop>showEmployeesDeleteButton</tabstop>
  <tabstop>showEmployeesRefreshButton</tabstop>
  <tabstop>treeView</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
